src/components/TransactionsTracker/useTransactionsTracker.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef } from 'react';
+import { getTransactionsByHashes as defaultGetTxByHash } from 'apiCalls/transactions';
+import { GetTransactionsByHashesType } from 'contexts/types';
+import { useCheckTransactionStatus } from 'hooks';
+import { useGetPendingTransactions } from 'services/transactions';
+import { getIsTransactionPending } from 'utils';
+
+export interface TransactionsTrackerType {
```

Comment:
Check the file above, same interface, declared twice. Probably should be lifted up or moved to types folder

Reply:
✅